bind/flag.go

```diff
@@ -190,10 +190,14 @@ func MarkFlagFilename(cmd *cobra.Command, names ...string) {
 func DescribeFlags(flags *pflag.FlagSet) string {
 	b := strings.Builder{}
 	flags.VisitAll(func(flag *pflag.Flag) {
-		if flag.Hidden || flag.Name == "help" {
+		if flag.Hidden || flag.Name == "help" || flag.Name == "verbose" {
 			return
 		}
-		b.WriteString(fmt.Sprintf("%s: %q\n", flag.Name, flag.Value))
+		s := flag.Value.String()
+		if s == "" {
```

Comment:
Let's just use empty value
https://www.educative.io/answers/how-to-represent-null-values-in-yaml